eo-maven-plugin/src/main/java/org/eolang/maven/dependencies/DcsEachWithoutTransitive.java

```diff
@@ -108,6 +109,16 @@ private static boolean eqTo(final Dependency left, final Dependency right) {
             && Objects.equals(left.getGroupId(), right.getGroupId());
     }
 
+    /**
+     * Is it our runtime dep?
+     * @param other The dep
+     * @return TRUE if it is
+     */
+    private static boolean isRuntime(final Dependency other) {
```

Comment:
@MikhailLipanin It looks like code duplication. We have the same code in `ResolveMojo`

Reply:
@volodya-lombrozo, Yes, but in `ResolveMojo` this method need to be `public static boolean` to be used in `DcsEachWithoutTransitive`.
So, as in `DcsWithRuntime` there is `private static boolean isRuntime(final Dependency other)`, I added the same code to `DcsEachWithoutTransitive`

Reply:
@MikhailLipanin let's make it `public`, it's much better comparing with code duplication as for me

Reply:
done